lib/baseFactory.js

```diff
@@ -210,6 +210,13 @@ class BaseFactory {
             });
     }
 
+    /**
+     * Cleanup
+     */
+    cleanUp() {
+        // no-op when not implemted by extender
```

Comment:
what would implement this method? example?

Reply:
so job factory cleanup method would be called by tasks set in the plugin. Added to base if any other factory classes intend to call this